### PR TITLE
Add Chinese Translation (2023.6.25)

### PR DESCRIPTION
**Pull request by SleepyAsh0191**

### CODE REVIEW

File: dataset/dictionary/y-events.json5 [hunk 1]
```diff
@@ -242,72 +249,83 @@
   //   tags: [ "event", "object" ],
   // },
   {
-    en: "water cannon",
+    en: "Water Cannon",
```
Comment (xicri) [inline dataset/dictionary/y-events.json5:252]: According to [v3.8 preview page](https://act.hoyoverse.com/ys/event/e20230624preview/index.html?game_biz=hk4e_global&hyl_presentation_style=fullscreen&hyl_landscape=true&hyl_hide_status_bar=true), w and c are lowercase. (See Secret Supper Paradise page > Park Attractions > Spino Blaster)

```suggestion
    en: "water cannon",
```

However, if there are any other official sources written as "Water Cannon" (with uppercase), uppercase may be preferrable. Did you see such sources?

File: dataset/dictionary/y-events.json5 [hunk 2]
```diff
@@ -242,72 +249,83 @@
...
     ja: "ウォーターキャノン",
+    // zhCN: "水炮",
```
Comment (xicri) [inline dataset/dictionary/y-events.json5:254]: Can you add a comment to explain why this is commented out?

**Review by xicri — CHANGES_REQUESTED** [review]
Welcome back and thanks as always!
I added two comments.

**Comment by xicri** [comment]
@Bill-Haku Can you review this PR?

File: dataset/dictionary/characters.json5 [hunk 3]
```diff
@@ -3312,6 +3324,8 @@
   {
     en: "Melusine",
     ja: "メリュジーヌ",
+    zhCN: "美露莘",
+    pinyins: [{ char: "莘", pron: "xin3" }],
```
Comment (Bill-Haku) [inline dataset/dictionary/characters.json5:3328]: The pronunciation of this character should be shen1 or xin1, as xin1 is used in names of locations, I think it should be pronounced as shen1

Ref:
image

File: dataset/dictionary/quests.json5 [hunk 4]
```diff
@@ -2265,7 +2266,8 @@
   },
   {
     en: "Ceaseless Training",
-    ja: "止まらぬ鍛錬",
+    ja: "止まらぬ鍛練",
```
Comment (Bill-Haku) [inline dataset/dictionary/quests.json5:2269]: 鍛錬 is more common in my experience. Have you confirmed it in official sources? @SleepyAsh0191 @xicri

**Review by Bill-Haku — CHANGES_REQUESTED** [review]

File: dataset/dictionary/quests.json5 (hunk 4 above)
Comment (xicri) [reply dataset/dictionary/quests.json5:2269]: @Bill-Haku Oops, yes, 鍛練 is correct.

Comment (xicri) [reply dataset/dictionary/quests.json5:2269]: @SleepyAsh0191 Can you include this suggestion?

File: dataset/dictionary/y-events.json5 (hunk 1 above)
Comment (SleepyAsh0191) [reply dataset/dictionary/y-events.json5:252]: i will change it back

File: dataset/dictionary/y-events.json5 (hunk 2 above)
Comment (SleepyAsh0191) [reply dataset/dictionary/y-events.json5]: I was ambiguous when I added the Chinese translation, and I just made a second confirmation

File: dataset/dictionary/quests.json5 (hunk 4 above)
Comment (SleepyAsh0191) [reply dataset/dictionary/quests.json5:2269]: I have added variants for this typo

Comment (xicri) [reply dataset/dictionary/quests.json5:2269]: Sorry, I just noticed I totally misunderstood the situation. Actually, no additional action was required for this.

I misunderstood @Bill-Haku said 鍛練 is correct and gave us a [Suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) to correct it, so I asked @SleepyAsh0191 to apply Bill's Suggestion.
Actually, Bill didn't send the Suggestion. I misunderstood the diff @SleepyAsh0191 made was Bill's Suggestion.

(@SleepyAsh0191 added 鍛錬 as a variant, but I think we can keep this.)

File: dataset/dictionary/characters.json5 (hunk 3 above)
Comment (Bill-Haku) [reply dataset/dictionary/characters.json5]: I also made a mistake. This character here is just used as the transliteration of the original English text, so xin1 is correct.

Comment (xicri) [reply dataset/dictionary/characters.json5]: @Bill-Haku OK, I understand xin1 is correct.

**Review by xicri — APPROVED** [review]

**Comment by xicri** [comment]
Thanks, this change should be deployed in a few minutes.
I will post updates on Twitter and Weibo tonight.

**Pull request merged**